changelog.md

```diff
@@ -2,6 +2,10 @@
 All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/) and follows [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
+## [1.XX.0] - 2023-XX-XX
```

Comment:
Can you remove the changes in the `changelog.md`? We will add this when bump the plugin to a new version